packages/components/src/IllustratedMessage/docs/image.tsx

```diff
@@ -0,0 +1,11 @@
+import { Content, Heading, IllustratedMessage, Image } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <IllustratedMessage>
```

Comment:
In the Image doc, maybe add objectFit like this? https://react-spectrum.adobe.com/react-spectrum/Image.html#object-fit